ui-modules/blueprint-composer/app/views/main/graphical/graphical.state.less

```diff
@@ -147,6 +147,8 @@
     &, .palette-full-height-wrapper {
       display: flex;
       flex-direction: column;
+      justify-content: space-between;
```

Comment:
prevents big space at the bottom when tiles are large.  does cause a little jumpiness when messages appear but i think it's the least bad answer at least without a lot of work.